prow/Makefile

```diff
@@ -56,8 +56,8 @@ ARTIFACT_UPLOADER_VERSION ?= $(TAG)
 NEEDS_REBASE_VERSION      ?= $(TAG)
 # CHECKCONFIG_VERSION is the version of the checkconfig image
 CHECKCONFIG_VERSION       ?= $(TAG)
-# GERRIT_CRIER_VERSION is the version of the gerrit-crier image
-GERRIT_CRIER_VERSION      ?= $(TAG)
+# CRIER_VERSION is the version of the crier image
+CRIER_VERSION      ?= $(TAG)
```

Comment:
is this in the prow cluster build bundle? I didn't see that

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/BUILD.bazel#L20, which got renamed here